accessing-data/webhooks-security.mdx

```diff
@@ -0,0 +1,136 @@
+---
+title: "Webhooks Security"
+description: "Learn how to implement signature verification for secure communication with webhooks"
+---
+
+When creating webhooks, you have the option to provide a secret token for added security. This secret token is used to generate a signature for each outgoing payload. To ensure the integrity of incoming webhook payloads, you can validate the signature on your server.
```

Comment:
Add a screenshot of the modal where you add a token, and a link to the webhooks page.